tests/Unit/Evolution/Systems/GeneralizedHarmonic/GhDampingFunctions/Test_GaussianPlusConstant.cpp

```diff
@@ -0,0 +1,115 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <cmath>
+#include <cstddef>
+#include <limits>
+#include <random>
+
+#include "DataStructures/DataVector.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/GhDampingFunctions/GaussianPlusConstant.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/GhDampingFunctions/GhDampingFunction.hpp"
+#include "Framework/SetupLocalPythonEnvironment.hpp"
+#include "Framework/TestCreation.hpp"
+#include "Framework/TestHelpers.hpp"
+#include "Helpers/Evolution/Systems/GeneralizedHarmonic/GhDampingFunctions/TestHelpers.hpp"
+#include "Parallel/PupStlCpp11.hpp"
+#include "Parallel/RegisterDerivedClassesWithCharm.hpp"
+#include "Utilities/Gsl.hpp"
+
+template <size_t VolumeDim, typename Fr>
```

Comment:
can you `cond` the forward decls? Doxygen is sometimes a bit silly and ignores are settings and decides to search files it shouldn't...